codalab/lib/beam/ratarmount.py

```diff
@@ -753,6 +753,7 @@ def _createIndex(
                     # fmt:on
                 )
             except tarfile.ReadError:
+                print("RE")
```

Comment:
is this the correct way to handle this? Might be ambiguous to the reader
```suggestion
                print("Tar read error")
```

Reply:
I meant to remove this. This was the original code from the ratarmount library and I just added it in for debugging.